### ISSUE TITLE
About contextual attention.

### ISSUE THREAD

**Issue by BlossomingL** [DAA233/generative-inpainting-pytorch#25]
Hi~ First, Your implemention is awesome, thank you! But I have some question.
1.What's the meaning of 'mm' in function 'contextual_attention'
2.I'am confused that after 'xi' and 'wi_normed' conv, then conv two times
It's kind of you if you could help me!

**Comment by DAA233** [comment]
Thanks for asking!
1. For the 'mm' in 'contextual_attention', I followed the same naming [here](https://github.com/JiahuiYu/generative_inpainting/blob/v1.0.0/inpaint_ops.py#L271).  From [this issue](https://github.com/JiahuiYu/generative_inpainting/issues/158),  'mm' indicates the background region after softmax.
2. Sorry, I am not sure what you mean that 'then conv two times'.

**Comment by herbiezhao** [comment]
看代码，一个batch只用了一个mask，如果一个image对应一个mask的话，代码是否需要修改下？
